docs/Login-View-Postmortem.md

```diff
@@ -0,0 +1,137 @@
+# Sign In View
```

Comment:
Is this file different from the one above?

Reply:
no, it was a copy of the branch README.md so it wouldn't get lost in a merge with master.  It's in a different folder now: docs/arch